models/edr/dbt_artifacts/alerts/alerts_dbt_models.sql

```diff
@@ -1,30 +0,0 @@
-{{
```

Comment:
This is a comment for the folders change above, why do we need this extra layer of hierarchy?

Reply:
This file was deleted.